server/apps/io/feeding_services/reuters.py

```diff
@@ -61,67 +46,38 @@ def _update(self, provider):
         if not last_updated or last_updated < updated - datetime.timedelta(minutes=ttl_minutes):
             last_updated = updated - datetime.timedelta(minutes=ttl_minutes)
 
-        for channel in self.get_channels():
-            for guid in self.get_ids(channel, last_updated, updated):
+        self.provider = provider
+        self.URL = provider.get('config', {}).get('url')
```

Comment:
Reason for moving `url` to `config` is there might be other http services which might have different URL and in future if reuters changes the url we don't want to touch the code. In any case the configuration is done at once.